packages/zowe-explorer/README.md

```diff
@@ -1,160 +1,161 @@
+
+To update securely stored user names and passwords in Zowe Explorer:
+
+1. Right click the profile and select **Manage Profile**.
+2. Select **Update Credentials** from the **Quick Pick**.
+
+    You are prompted for the new credentials and these are saved to the secure credentials vault.
+
+## Other authentication methods
+
+Zowe Explorer for VS Code supports multiple authentication methods including basic authentication, multi-factor authentication, tokens, and certificates.
+
+### Using Single Sign-On
+
+As a Zowe user, you can use a base profile stored in a team configuration file to access multiple services through Single Sign-On.
+
+In Zowe Explorer, a base profile enables you to authenticate your credentials with one method, the Zowe API Mediation Layer (API ML), to access multiple services. For more information, see [profile types](https://docs.zowe.org/stable/user-guide/cli-using-using-team-profiles#zowe-cli-profile-types).
+
+To log into the API ML authentication service with an existing base profile:
+
+1. Right-click on the profiles you want to connect through with the API ML.
+2. Select the **Manage Profile** option from the context menu.
+3. In the **Quick Pick**, select **Log in to Authentication Service**.
+4. In the following **Quick Pick** menu, select the appropriate option for authenticating to the API ML.
+5. Answer the proceeding prompts for information.
+
+    If the request is successful, the token is used for authentication until the logout action is taken or the token expires.
 
 For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-using-integrating-apiml).
 
-If you do not want to store your token, you can request the server to end your session token. Use the **Log out from Authentication Service** feature to invalidate the token:
+If you are done working with Zowe Explorer and want to prevent further use of a token, you can request the server to invalidate your session token. Use the **Log out from Authentication Service** feature to invalidate the token:
 
 1. Open Zowe Explorer.
-1. Hover over **DATA SETS**, **USS**, or **JOBS**.
-1. Click the **+** icon.
-1. Right-click your profile and select **Manage Profile**.
-1. Select the **Log out from Authentication Service** option.
+2. Right-click your profile.
+3. Select the **Manage Profile** option.
+4. In the **Quick Pick**, select the **Log out from Authentication Service** option.
 
-Your token has been successfully invalidated.
+    Your token has been successfully invalidated.
 
-## Usage tips
+### Multi-factor authentication (MFA) support
 
-- Use the **Add to Favorite** feature to permanently store chosen data sets, USS files, and jobs in the **Favorites** folder. Right-click on a data set, USS file or jobs and select **Add Favorite**.
+Zowe Explorer supports the use of MFA through the Zowe API Mediation Layer (API ML).
```

Comment:
same with this section on support for zosmf profiles, check with extenders for their support in these scenarios